src/main/java/iudx/data/ingestion/server/authenticator/model/JwtData.java

```diff
@@ -6,7 +6,7 @@
 @DataObject(generateConverter = true, publicConverter = false)
 public final class JwtData {
 
-  private String access_token;
+  private String accessToken;
```

Comment:
call `setAccessToken()` from `JwtData(JsonObject json)`  as done in RS [here](https://github.com/datakaveri/iudx-resource-server/pull/426/files#diff-b21ebe79a0d682898e2a4b849071d35fe264a4154823384bcc12a6cd0ee21f38R25).

Reply:
done